api/backend.go

```diff
@@ -7,6 +7,7 @@ import (
 	"math/big"
 	"sync"
 
+	"encoding/hex"
```

Comment:
Ideally it should be put in a group above where all built-in libraries are listed.